lib/stdlib/src/re.erl

```diff
@@ -19,6 +19,7 @@
 %%
 -module(re).
 -export([grun/3,urun/3,ucompile/2,replace/3,replace/4,split/2,split/3]).
+-export_type([mp/0]).
```

Comment:
I am not really sure about this one. In fact, I think the whole idea is to make the type opaque.

Reply:
good idea, dropping re:mp